src/TestBundle/TestBundle.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace TestBundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+class TestBundle extends Bundle
```

Comment:
i think at some point, the idea was to have some demo things but looking at we have in this bundle vs the other one, i feel we could just move all to the AppBundle.